### PR TITLE
change external sync order for serializers

### PR DESCRIPTION
**Pull request by costaconrado**
This PR changes the order that serializers are called in Flaw API.

This PR also modifies `BugzillaSyncMixinSerializer` so it delegates creation to `serializers.ModelSerializer.create` method, this allow all validations to run before syncing to Bugzilla and also works with many-to-many models.
This PR also modifies `BugzillaSyncMixinSerializer.update` and  `AffectView.bulk_put` methods in order to skip `bzsync` model method, avoiding saving flaw multiple times during bulk operation.

Closes OSIDB-3029.

### CODE REVIEW

**Review by osoukup — APPROVED** [review]
Looks much better

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed Flaw CC list builder to generate CCs in Bugzilla format
   for both Bugzilla and Jira tracked PS modules (OSIDB-2985)
 - Flaw commments create action respects is_private (OSIDB-3003)
+- Fix external sync order in serializers (OSIDB-3029)
```
Comment (jobselko) [inline docs/CHANGELOG.md:27]: Not in `## Unreleased` section

Comment (costaconrado) [reply docs/CHANGELOG.md:27]: nice catch, sorry about that.

**Review by jobselko — APPROVED** [review]
"Enforced" approve as I do not fully understand the changes :)

**Review by MrMarble — APPROVED** [review]
I don't know what this does but seems to be well written 😄

**Pull request merged**